### ISSUE TITLE
EV3 debug logging

### ISSUE THREAD

**Issue by cschol** [squinkylabs/SquinkyVCV#35]
Minor issue: EV3 shows debug logging when changing waveforms.

For example:

```
set curValue=1.000000
set curValue=1.000000
set curValue=4.000000
set curValue=0.000000
set curValue=0.000000
set curValue=3.000000
set curValue=0.000000
set curValue=5.000000
set curValue=0.000000
```

**Comment by squinkylabs** [comment]
Fixed in master now. Probably won't trigger a new build just for this, but will keep the issue open until a fix is deployed to the plugin manager.

**Comment by squinkylabs** [comment]
Fixed in 0.6.10
